docs/changelog/1_0_b2.rst

```diff
@@ -0,0 +1,102 @@
+  migrations (:eql:gh:`#2328`).
+* Fix tuples in ``MULTI`` properties and :eql:op:`IN` expressions
+  (:eql:gh:`#2255`).
+* Fix SDL handling of types with two or more indexes
+  (:eql:gh:`#2301`).
+* Fix a regression with array aliasing that made array aliases
+  indistinguishable (:eql:gh:`#2287`).
+* Fix a number of issues with constraints and provide
+  better context information for constraint definition errors
+  (:eql:gh:`#1370`, :eql:gh:`#2250`, :eql:gh:`#2305`, :eql:gh:`#2307`,
+  :eql:gh:`#2311`, :eql:gh:`#2410`).
+* Fix ``DROP OWNED`` on links and properties with defaults
+  (:eql:gh:`#2306`).
+* Fix ``SET TYPE`` on links and properties with constraints
+  (:eql:gh:`#2309`).
+
+
+EdgeQL
+======
+
+* Drop the deprecated ``Port``. The more general
+  :ref:`extension <ref_datamodel_extensions>` mechanism introduced in
+  EdgeDB 1.0 beta 1 should be used (:eql:gh:`#2262`).
+* Reduce the maximum length for names of databases and roles to 51
+  characters (:eql:gh:`#2465`).
+* Enable ``DESCRIBE SCHEMA AS SDL`` (:eql:gh:`#2481`).
+* Support ``UNLESS CONFLICT ON`` with two or more properties or links
+  (:eql:gh:`#1939`).
+* Make :eql:func:`min` and :eql:func:`max` work more consistently
+  across all supported types (:eql:gh:`#1920`).
+* Improve cardinality inference (:eql:gh:`#2097`).
+* Disallow use of ``VOLATILE`` functions in schema-defined computable
+  expressions (:eql:gh:`#2467`).
+* Fix handling of collection types of non-builtin scalars in dumps
+  (:eql:gh:`#2349`).
+* Fix inconsistent handling of ``{}`` by the :eql:op:`IF` operator
+  (:eql:gh:`#2460`).
+* Fix duplicate values appearing when using :ref:`+=
+  <ref_eql_statements_update>` (:eql:gh:`#2455`).
+* Fix an issue with empty sets (i.e. ``{}``) inside set literals
+  (:eql:gh:`#2154`).
+* Fix backward links when multiple types with the same link name exist
+  (:eql:gh:`#2360`).
+* Fix :eql:op:`DISTINCT` on empty and nested tuples (:eql:gh:`#2333`).
+* Fix some serialization issues of shapes inside arrays and tuples
+  (:eql:gh:`#1818`).
+* Make sure :ref:`DELETE <ref_eql_statements_delete>` also applies to
+  all sub-types of the selected type (:eql:gh:`#2265`).
+
+GraphQL
+=======
+
+* Fix usage of :eql:type:`enums <enum>` as input variables in GraphQL
+  (:eql:gh:`#2415`).
+* Fix querying ``BaseObject`` via GraphQL (:eql:gh:`#2214`).
+
+
+Command-Line Tools
+==================
+
+We've added ``edgedb project init`` command to help manage EdgeDB
+credentials for your project by using instance names rather than with
+more verbose command options. This instance name can be used in CLI as
+well as programmatically when connecting to EdgeDB.
+
+
+Bindings
+========
+
+* Implement `RFC 1004 <robust_>`_ features for `edgedb-go
+  <https://github.com/edgedb/edgedb-go>`_ driver.
+* Update the `edgedb-python
+  <https://github.com/edgedb/edgedb-python>`_ driver to v0.14.0.
+* Update the `edgedb-js <https://github.com/edgedb/edgedb-js>`_ driver
+  to v0.14.0.
```

Comment:
I think we should also mention Deno support. @jaclarke can you contribute a short hello world 10 lines example to mention that properly?

Reply:
Added the release of Deno to the list.